packages/design-system/src/styles/utilities/_display-visibility.scss

```diff
@@ -9,6 +9,10 @@
   display: inline-block !important;
 }
 
+.ds-u-display--inline {
```

Comment:
Usually we don't want to add an additional utility for this type of ticket, however in this case it seems to make sense. Because we are adding, we should also update the documentation and examples for the display/visibility utility page

Reply:
Good pick up. Done.